src/components/EditorContainer.js/EditorContainer.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import styles from "./editor-container.module.css";
+
+import { CKEditor } from "@ckeditor/ckeditor5-react";
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+
+import { useNotesContext } from "../../context";
+
+import { FaTags } from "react-icons/fa";
+import { MdOutlinePriorityHigh } from "react-icons/md";
+import { AiTwotoneEdit } from "react-icons/ai";
+
+const priorities = [
+  { id: 1, name: "Low" },
+  { id: 2, name: "Medium" },
+  { id: 3, name: "High" },
+];
+
+export const EditorContainer = () => {
+  const {
+    notesState: { tags },
+  } = useNotesContext();
+
+  return (
+    <div className={styles.editorContainer}>
+      <div className={styles.titleContainer}>
+        <AiTwotoneEdit />
+        <input className={styles.title} />
```

Comment:
In this `input`, you have not given it any `state` for `value`. So how are you getting/retrieving it's value ?

Reply:
I have worked on it in the future branches. Since reviews are slow, I have worked on branches with different functionalities. Sorry for the confusion.